airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -136,14 +134,14 @@ def read(
 
     @property
     def per_stream_state_enabled(self):
-        return False  # While CDK per-stream is in active development we should keep this off
+        return True  # While CDK per-stream is in active development we should keep this off
 
     def _read_stream(
         self,
         logger: logging.Logger,
         stream_instance: Stream,
         configured_stream: ConfiguredAirbyteStream,
-        connector_state: MutableMapping[str, Any],
+        connector_state: ConnectorStateManager,
```

Comment:
Since these are intended to be private methods, we should be okay to modify the interface since this is only intended to be invoked from within `read()`. I did a search through existing integrations and didn't see any source connectors overriding this (which they really shouldn't be).